toolchain/gcc/common.mk

```diff
@@ -89,7 +89,7 @@ endif
 
 GCC_CONFIGURE:= \
 	SHELL="$(BASH)" \
-	$(if $(shell gcc --version 2>&1 | grep LLVM), \
+	$(if $(shell gcc --version 2>&1 | grep -E "Apple.(LLVM|clang)"), \
```

Comment:
Why are you making it Apple specific?

Reply:
I just follow by this line: https://github.com/openwrt/openwrt/blob/master/include/prereq-build.mk#L40